routes/login/login.controller.js

```diff
@@ -133,3 +235,135 @@ const postLogin = async (req, res, next) => {
 
   next()
 }
+
+const postAge = (req, res, next) => {
+  const ageYesNo = req.body.ageYesNo
+
+  req.session.login.ageYesNo = ageYesNo
+
+  next()
+}
+
+const postTaxableIncome = (req, res, next) => {
+  const taxableIncome = req.body.taxableIncome
+
+  req.session.login.taxableIncome = taxableIncome
+
+  if (taxableIncome !== 'Yes') {
+    return res.redirect('/offramp/taxable-income')
+  }
+
+  next()
+}
+
+const postResidenceScreening = (req, res, next) => {
+  const residenceScreening = req.body.residenceScreening 
+
+  req.session.login.residenceScreening  = residenceScreening 
+
+  if (residenceScreening  === 'No') {
+    return res.redirect('/offramp/residence')
+  }
+
+  next()
+}
+
+const postChildren = (req, res, next) => {
+  const children = req.body.children
+
+  req.session.login.children = children
+
+  if (children !== 'No') {
+    return res.redirect('/offramp/children')
+  }
+
+  next()
+}
+
+const postEligibleDependents = (req, res, next) => {
+  const eligibleDependents = req.body.eligibleDependents
+
+  req.session.login.eligibleDependents = eligibleDependents
+
+  if (eligibleDependents === 'No') {
+    return res.redirect('/eligibility/tuition')
+  }
+
+  if (eligibleDependents === 'Yes') {
+    return res.redirect('/eligibility/dependents-claim')
+  }
+
+  next()
+}
+
+const postEligibleDependentsClaim = (req, res, next) => {
+  const eligibleDependentsClaim = req.body.eligibleDependentsClaim
+
+  req.session.login.eligibleDependentsClaim = eligibleDependentsClaim
+
+  if (eligibleDependentsClaim === 'No') {
+    return res.redirect('/eligibility/tuition')
+  }
+
+  if (eligibleDependentsClaim === 'Yes') {
+    return res.redirect('/offramp/dependents')
+  }
+
+  next()
+}
+
+const postTuition = (req, res, next) => {
+  const tuition = req.body.tuition
+
+  req.session.login.tuition = tuition
+
+  if (tuition === 'No') {
+    return res.redirect('/eligibility/income-sources')
+  }
+
```

Comment:
for these ones where we run an `if NO` _and_ `if YES`, I assume one of the redirects we've put in the hidden redirect input, yes? So in this example, I think the NO url is in the redirect input, so we only need to check if tuition === YES and redirect. Otherwise, carry on with `next()` which should take to doRedirect.
Unless I've completely misunderstood, in which case lemme know!